src/main/java/net/minecraftforge/client/gui/ScrollPanel.java

```diff
@@ -48,12 +47,22 @@
     private boolean scrolling;
     protected float scrollDistance;
     protected boolean captureMouse = true;
-    protected final int border = 4;
+    protected final int border;
 
-    private final int barWidth = 6;
+    private final int barWidth;
     private final int barLeft;
+    private final int bgColorFrom;
+    private final int bgColorTo;
+    private final int barBgColor;
+    private final int barColor;
+    private final int barBorderColor;
 
     public ScrollPanel(Minecraft client, int width, int height, int top, int left)
+    {
+        this(client, width, height, top, left, 4, 6, 0xC0101010, 0xD0101010, 0xFF000000, 0xFF808080, 0xFFC0C0C0);
+    }
+
+    public ScrollPanel(Minecraft client, int width, int height, int top, int left, int border, int barWidth, int bgColorFrom, int bgColorTo, int barBgColor, int barColor, int barBorderColor)
```

Comment:
This constructor seems a bit too redundant if seven new parameters are being added. You may want to create other constructors that just modify a single section (such as change background color, change bar info, etc.)